app/src/main/java/com/keylesspalace/tusky/components/search/SearchViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package com.keylesspalace.tusky.components.search
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModel
+import androidx.paging.PagedList
+import com.keylesspalace.tusky.components.search.adapter.SearchRepository
+import com.keylesspalace.tusky.entity.Account
+import com.keylesspalace.tusky.entity.HashTag
+import com.keylesspalace.tusky.entity.Status
+import com.keylesspalace.tusky.network.MastodonApi
+import com.keylesspalace.tusky.util.Listing
+import com.keylesspalace.tusky.util.NetworkState
+import com.keylesspalace.tusky.util.ViewDataUtils
+import com.keylesspalace.tusky.viewdata.StatusViewData
+import io.reactivex.disposables.CompositeDisposable
+import javax.inject.Inject
+
+class SearchViewModel @Inject constructor(private val mastodonApi: MastodonApi) : ViewModel() {
```

Comment:
```suggestion
class SearchViewModel @Inject constructor(private mastodonApi: MastodonApi) : ViewModel() {
```

Reply:
It was left for further updates, but it's not useful now.
Updated